tests/D2L.CodeStyle.Analyzers.Test/Specs/ImmutabilityAnalyzer.cs

```diff
@@ -1355,6 +1360,36 @@ public sealed record
 		/* MissingTransitiveImmutableAttribute(ConsistencyTests.DerivedRecordNoAttrWithArg, , base class, ConsistencyTests.BaseRecordWithArgs) */ DerivedRecordNoAttrWithArg /**/
 		( int z ) : BaseRecordWithArgs( z );
 
+	public sealed record class
+		/* MissingTransitiveImmutableAttribute(ConsistencyTests.DerivedExplicitRecordNoAttrConstArg, , base class, ConsistencyTests.BaseRecordWithArgs) */ DerivedExplicitRecordNoAttrConstArg /**/
+		: BaseRecordWithArgs( 0 );
+
+	public sealed record class
+		/* MissingTransitiveImmutableAttribute(ConsistencyTests.DerivedExplicitRecordNoAttrWithArg, , base class, ConsistencyTests.BaseRecordWithArgs) */ DerivedExplicitRecordNoAttrWithArg /**/
+		( int z ) : BaseRecordWithArgs( z );
+
+
+	[Immutable]
+	public readonly record struct ReadOnlyRecordStruct() { }
+
+	[Immutable]
+	public readonly record struct ReadOnlyRecordStructWithArg( int x ) { }

```

Comment:
`readonly record struct` defaults props to `{ get; init; }`. Without `readonly` it's `{ get; set; }`